src/workspace/clinical-lowlight-theme/TabDropDown.module.scss

```diff
@@ -3,7 +3,7 @@
     --terra-application-workspace-tab-drop-down-border: 1px solid transparent;
     --terra-application-workspace-tab-drop-down-border-radius: 3px;
     --terra-application-workspace-tab-drop-down-max-height: 49px;
-    --terra-application-workspace-tab-drop-down-max-width: 187px;
+    --terra-application-workspace-tab-drop-down-max-width: 187;;
```

Comment:
no px and double ';'? seems like a typo.

Reply:
Updated.